dotcom-rendering/src/components/LiveBlogBlocksAndAdverts.tsx

```diff
@@ -22,6 +23,17 @@ type Props = {
 	isInLiveblogAdSlotTest?: boolean;
 };
 
+const appsFirstAdIndex = 1;
+
+/**
+ * On apps, add an {@linkcode AdPlaceholder} after the 2nd block, and then
+ * after every 5th block from then on.
+ */
+const shouldInsertAppAd = (isApps: boolean, blockIndex: number): boolean =>
+	isApps &&
+	(blockIndex === appsFirstAdIndex ||
+		(blockIndex - appsFirstAdIndex) % 5 === 0);
```

Comment:
I think there are a couple of reasons for wanting to try this:

1. We attempt to minimise the impact on ad metrics with the transition to DCR, by inserting the same number of ads in the same positions as on existing apps blogs.
2. Commercial's have two insertion algorithms, one for narrower breakpoints ("mobile") and one for wider ones ("desktop"), to reflect the fact that the layouts are different. We are currently using the narrower algorithm even for wider breakpoints on DCR for apps articles, because apps ads are different enough to make it difficult to use the full Commercial implementation.

See also @arelra 's comment here: https://github.com/guardian/dotcom-rendering/pull/8843#discussion_r1338470886

We don't have to make this change however if we don't think it's the right approach! Happy to discuss with you and @arelra ?